docker/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.5'
+
+services:
+
+  frontend:
+    build:
+      context: ../
+      dockerfile: ./Dockerfile
+    ports:
+      - '3000:3000'
+    profiles:
+      - default
+      - all
+
+  frontend-frontier:
+    build:
+      context: ../
+      dockerfile: ./Dockerfile.frontier
+    ports:
+      - '3001:3000'
+    profiles:
+      - frontier
+      - all
+
+  frontend-testnet:
+    build:
+      context: ../
+      dockerfile: ./Dockerfile.testnet
+    ports:
+      - '3002:3000'
+    profiles:
+      - testnet
+      - all
```

Comment:
What is the compose file used for? What scenario would we run all these services together?

Reply:
They are all mutually exclusive unless you select the profile `all`.

By the default it runs the main frontend but you can select frontier or testnet.

If you want, you could run them all together by selecting the all profile. It could be used in some testing scenarios.